.github/workflows/CI.yml

```diff
@@ -8,8 +8,8 @@ on:
   workflow_dispatch:
 
 jobs:
-  build-and-deploy:
-    runs-on: ubuntu-20.04
+  BuildAndDeploy:
+    runs-on: ubuntu-latest
```

Comment:
Why `ubuntu-latest`?

Reply:
No reason in particular. Just so that it's more future proof. In my experience, both work flawless.

Reply:
I find the opposite 😂 when you use a OS that you don't know how it will perform in the future is less future proof. Using ubuntu-20.04 will break but you can have more control and no surprises.

Reply:
You're right. I've set both jobs to ubuntu-20.04.